sdk/managednetworkfabric/Azure.ResourceManager.ManagedNetworkFabric/CHANGELOG.md

```diff
@@ -1,20 +1,21 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.1.1 (2023-11-29)
```

Comment:
should this be 1.2.0?

Reply:
A minor version bump occurs only when the api-version is upgraded.